src/net_processing.cpp

```diff
@@ -2454,6 +2452,9 @@ void ProcessMessage(
             // Get recent addresses
             connman.PushMessage(&pfrom, CNetMsgMaker(nSendVersion).Make(NetMsgType::GETADDR));
             pfrom.fGetAddr = true;
+
+            // Moves address from New to Tried table in Addrman
+            // (unrelated to advertising our address)
```

Comment:
I feel strongly that we should not cement opinions into docs in the codebase. since there are divergent opinions on whether or not we should have this logic here, I'd like to avoid documentation that indicates it must be here.

Reply:
I don't find this comment particularly helpful... Why it is done here, why for outbounds and non-blocks-only?

Reply:
resolving this conversation just to close off threads, feel free to continue [over here](https://github.com/bitcoin/bitcoin/pull/19724#discussion_r482610113)